modulefiles/module_base.orion

```diff
@@ -12,47 +12,49 @@ module load contrib noaatools
 ## load programming environment
 ## this typically includes compiler, MPI and job scheduler
 ##
+# Core modules
 module load intel/2018
 module load impi/2018
 
-#For ocean post: 
 module load ncl/6.6.2
 module load nco/4.8.1
 
-##
-## NCEP libraries (temporary version to match the CCPP requirements)
-##
+module load slurm/19.05.3-2
+# Stack modules
+
+module use /apps/contrib/NCEP/libs/modulefiles/stack
+module load hpc/1.0.0
+module load hpc-intel/2018.4
+module load prod_util/develop
+#module load hpc-impi/2018.4
+module load bacio
```

Comment:
Can we leave the version numbers on here? Also, do these match with the model that's being built?